de.tu_bs.cs.isf.e4cf.text_editor/src/de/tu_bs/cs/isf/e4cf/text_editor/view/TextEditor.java

```diff
@@ -0,0 +1,661 @@
+package de.tu_bs.cs.isf.e4cf.text_editor.view;
+
+import java.net.URL;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.ResourceBundle;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.regex.Matcher;
+
+import javax.inject.Inject;
+import org.apache.commons.lang3.StringUtils;
+
+import de.tu_bs.cs.isf.e4cf.core.util.RCPMessageProvider;
+import de.tu_bs.cs.isf.e4cf.core.util.ServiceContainer;
+import de.tu_bs.cs.isf.e4cf.text_editor.FileUtils;
+import de.tu_bs.cs.isf.e4cf.text_editor.stringtable.EditorST;
+import javafx.application.Platform;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.control.Alert.AlertType;
+import javafx.scene.control.IndexRange;
+import javafx.scene.control.Label;
+import javafx.scene.control.Menu;
+import javafx.scene.control.MenuItem;
+import javafx.scene.control.Tab;
+import javafx.scene.control.TabPane;
+import javafx.scene.input.Clipboard;
+import javafx.scene.input.ClipboardContent;
+import javafx.scene.input.DataFormat;
+import org.fxmisc.richtext.CodeArea;
+import org.fxmisc.richtext.LineNumberFactory;
+import org.fxmisc.richtext.model.StyleSpans;
+import org.fxmisc.richtext.model.StyleSpansBuilder;
+
+/**
+ * 
+ * @author Soeren Christmann, Cedric Kapalla, Lukas Cronauer, Erwin Wijaya
+ *
+ */
```

Comment:
Die Klasse ist halt echt ziemlich groß. Ich denke, ihr solltet eure View in Mehren FXML Files mit jeweiligen Controllern oder so schreiben damit das alles etwas übersichtlicher wird, bzw. ich weiß nicht, ob das so geht aber ein Versuch ist es sicherlich wert.